src/components/item/item.ios.scss

```diff
@@ -210,6 +210,17 @@ $item-ios-sliding-content-background:     $list-ios-background-color !default;
     background-position: right ($item-ios-padding-right - 2) center;
     background-size: 14px 14px;
   }
+  [dir="rtl"] .item-ios[detail-push] .item-inner,
```

Comment:
Dismiss this comment if you think otherwise. (it is more of a preference, not style)

I think that:
- There should be a break-line between 212 and 213
- Instead of having:
```
[dir="rtl"] .item-ios[detail-push] .item-inner,
[dir="rtl"] button.item-ios:not([detail-none]) .item-inner,
[dir="rtl"] a.item-ios:not([detail-none]) .item-inner {
  ...
}
```

You should have:
```
[dir="rtl"] {
  .item-ios[detail-push] .item-inner,
  button.item-ios:not([detail-none]) .item-inner,
  a.item-ios:not([detail-none]) .item-inner {
   ...
  }
}

```